package.json

```diff
@@ -7,19 +7,25 @@
   "license": "MIT",
   "scripts": {
     "start": "nodemon src/app.js",
-    "test": "yarn validate && jest --passWithNoTests",
+    "test": "yarn validate && jest --detectOpenHandles --forceExit --verbose",
```

Comment:
why you didnt add IS_JEST=1 here?

Reply:
I added IS_JEST=1 to `.env` file. Since `.env` is not tracked by git, you cannot see it.